gensrc/thrift/MasterService.thrift

```diff
@@ -44,6 +44,7 @@ struct TTabletInfo {
     13: optional Types.TPartitionId partition_id
     14: optional bool is_in_memory
     15: optional bool enable_persistent_index
+    16: required Types.TVersion min_readable_version
```

Comment:
You can also change the TVersionHash to TVersion?

Reply:
What do you mean? Why change TVersionHash to TVersion, it's a bad practice to change the existing field

Reply:
TVersionHash is already not used also from 1.9. I think we can remove it thoroughly.

Reply:
ok